frontend/docsettings.lua

```diff
@@ -261,7 +276,9 @@ function DocSettings:open(doc_path)
     else
         new.data = {}
     end
-    new.data.doc_path = doc_path
+    if new.data.doc_path ~= "<ignore>" then
```

Comment:
> I added a check that if the existing metadata doc_path equals <ignore> then it isn't overwritten.

I still don't understand why you need this with your syncing stuff.
Many other things change, that you should ignore - so why not ignore (on your syncing side) this one.
Not fond of adding unrelated/unexplainable stuff on our side without some logic/comment/use-case explanation.

Reply:
It's true that other things change, this is just the one that seemed to change far more than the others. But I'll get rid of it here, doesn't need to be a part of this PR.